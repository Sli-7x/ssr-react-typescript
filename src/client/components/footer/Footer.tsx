import * as React from 'react';
import styled from 'styled-components';

const Text = styled.span`
  color: red;
`;

const date = () => {
  return new Date().getFullYear();
};

const Footer = () => (
  <div id="footer">
    <Text>&copy; {date()}</Text>
  </div>
);

export default Footer;
