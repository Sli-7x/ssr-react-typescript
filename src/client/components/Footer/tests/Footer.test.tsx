import * as React from 'react';
import { shallow } from 'enzyme';
import Footer from '../Footer';
import * as renderer from 'react-test-renderer';

describe('Footer.tsx', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
