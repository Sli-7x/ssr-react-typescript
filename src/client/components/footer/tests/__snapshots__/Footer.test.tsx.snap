// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer.tsx should render correctly 1`] = `
<div
  id="footer"
>
  <span
    className="sc-bdVaJa rOCEJ"
  >
    Â© 
    2018
  </span>
</div>
`;
